src/elarm_event.erl

```diff
@@ -72,7 +72,7 @@ unsubscribe(Ref, #evt_state{ subs = Subs } = State)->
 
 -spec new_alarm(alarm(), #evt_state{}) -> {ok, #evt_state{}} | {error, term()}.
```

Comment:
The spec in line 73 uses a record, please define a type for the record and use that instead.